packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/repository.test.ts

```diff
@@ -338,6 +354,17 @@ describe('SavedObjectsRepository', () => {
           expect.anything()
         );
       });
+      // this test only ensures that the client accepts the managed field in a document
+      it(`formats the ES request with managed=true in a document`, async () => {
+        const obj1WithManagedTrue = { ...obj1, managed: true };
+        const obj2WithManagedTrue = { ...obj2, managed: true };
+        await bulkCreateSuccess(client, repository, [obj1WithManagedTrue, obj2WithManagedTrue]);
+        const body = [...expectObjArgs(obj1WithManagedTrue), ...expectObjArgs(obj2WithManagedTrue)];
+        expect(client.bulk).toHaveBeenCalledWith(
+          expect.objectContaining({ body }),
+          expect.anything()
+        );
+      });
```

Comment:
For this test and the `should use the ES index method...` above I think we are not gaining new coverage since we already are testing that `body` is being passed to `client.bulk` (where `managed` is set on `obj1|obj2`).

If we want to cover behaviour for managed `true | false` maybe we